specification/search/resource-manager/Microsoft.Search/preview/2024-03-01-preview/search.json

```diff
@@ -0,0 +1,3298 @@
+              },
+              {
+                "value": "Deleting",
+                "name": "Deleting",
+                "description": "The private link service connection is in the process of being deleted."
+              },
+              {
+                "value": "Failed",
+                "name": "Failed",
+                "description": "The private link service connection has failed to be provisioned or deleted."
+              },
+              {
+                "value": "Succeeded",
+                "name": "Succeeded",
+                "description": "The private link service connection has finished provisioning and is ready for approval."
+              },
+              {
+                "value": "Incomplete",
+                "name": "Incomplete",
+                "description": "Provisioning request for the private link service connection resource has been accepted but the process of creation has not commenced yet."
+              },
+              {
+                "value": "Canceled",
+                "name": "Canceled",
+                "description": "Provisioning request for the private link service connection resource has been canceled."
+              }
+            ]
+          }
+        }
+      },
+      "description": "Describes the properties of an existing Private Endpoint connection to the search service."
+    },
+    "PrivateEndpointConnectionListResult": {
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateEndpointConnection"
+          },
+          "description": "The list of Private Endpoint connections."
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Request URL that can be used to query next page of private endpoint connections. Returned when the total number of requested private endpoint connections exceed maximum page size."
+        }
+      },
+      "description": "Response containing a list of Private Endpoint connections.",
+      "x-ms-external": true
+    },
+    "SemanticSearch": {
+      "type": "string",
+      "enum": [
+        "disabled",
+        "free",
+        "standard"
+      ],
+      "x-ms-enum": {
+        "name": "SearchSemanticSearch",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "disabled",
+            "name": "disabled",
+            "description": "Indicates that semantic search is disabled for the search service. This is the default."
+          },
+          {
+            "value": "free",
+            "name": "free",
+            "description": "Enables semantic ranking on a search service and indicates that it is to be used within the limits of the free tier. This would cap the volume of semantic ranking requests and is offered at no extra charge. This is the default for newly provisioned search services."
```

Comment:
instead of "free tier", could you say "free plan".  Then you could reinforce by starting the next sentence with the 'Free plan caps the volume of ..."   Lots of customer confusion around free tier/plan verbiage so the reinforcement helps dial that in.